KubeArmor/core/kubeArmor.go

```diff
@@ -279,7 +280,7 @@ func GetOSSigChannel() chan os.Signal {
 // ========== //
 
 // KubeArmor Function
-func KubeArmor(clusterName, gRPCPort, logPath, logFilter string, enableHostPolicy, enableEnforcerPerPod bool) {
+func KubeArmor(clusterName, gRPCPort, logPath, logFilter string, enableHostPolicy, enableEnforcerPerPod bool, enableKvmAgent bool) {
```

Comment:
can you update these on the latest code?

Reply:
Sure @nam-jaehyun .
Working on this review comment.